cl/people_db/management/commands/cl_import_judges.py

```diff
@@ -88,7 +102,11 @@ def import_fjc_judges(self, infile=None):
         df['Professional Career'].replace(to_replace=r';\sno', value=r', no',
                                             inplace=True, regex=True)
         for i, row in df.iterrows():
-            make_federal_judge(dict(row), testing=self.debug)
+            if i < self.options['offset']:
+                make_federal_judge(dict(row), testing=self.debug)
```

Comment:
This isn't quite right. You `continue` if it's less than the offset, and if it's more, you do the work. 

So if the offset is 5, you skip the first five records (i.e., you `continue` until `i` is 6, *then* you do the work. If this doesn't make sense, grep the code for the word "offset" and you should see a handful of additional examples.

Reply:
Just corrected and resubmitted this.